protocols/manage/count.go

```diff
@@ -106,9 +106,11 @@ func (p *ProtocolCount) FuncPC() {
 		p.SendTo(p.Parent(), &NodeIsUp{})
 	}
 	if !p.IsLeaf() {
-		for _, c := range p.Children() {
-			dbg.Lvl3(p.Info(), "sending to", c.Entity.Addresses, c.Id, p.timeout)
-			p.SendTo(c, &PrepareCount{Timeout: p.timeout})
+		for _, child := range p.Children() {
+			go func(c *sda.TreeNode) {
```

Comment:
Just click on "Show 2 comments":

time of creation for connections - usually the count-protocol is used when no connections are set up yet. If it has to wait for all connections to be set up in a serial way, the timeout is reached too fast. Used in simulations where there are more than 2000 nodes involved.